day_6/exercises/burrito.rb

```diff
@@ -17,3 +18,13 @@ def initialize(protein, base, toppings)
 p dinner.protein
 p dinner.base
 p dinner.toppings
+
+#Add topping
+dinner.toppings << 'Jalapenos'
+p dinner.toppings
+# remove topping
+dinner.toppings.delete_at(2)
+p dinner.toppings
+# change_protein
+dinner.protein = "Steak"
+p dinner.protein
```

Comment:
Clever use of the `attr_accessor` methods combined with Ruby methods to achieve the state change to `dinner`. Try re-writing these as methods within your `Burrito` class and then calling the methods to achieve the same output.